### ISSUE TITLE
Add track opens and clicks

### ISSUE THREAD

**Issue by ssoulless** [Sology/maily_herald#3]
Hello, I think this is very important for an email marketing campaign.

Know how many people you sent the email opened, and also track clicks

Thank you ! I want to know if you are planning to add those features, that would make maily_herald more close to mailchimp.

Regards

**Comment by ljachymczyk** [comment]
Yes, we plan to include this in the releases after 0.9 which is coming soon.

**Comment by cromulus** [comment]
Any updates?
